index.html

```diff
@@ -97,16 +97,19 @@ <h2 id="docconformance">
         requirements for the use of ARIA markup in HTML documents and describes
         the <a>implicit ARIA semantics</a> that apply to <a data-cite=
         "html/dom.html#elements">HTML elements</a> as defined in the HTML
-        Accessibility API Mappings 1.0 [[html-aam-1.0]] specification. Each
-        language feature (element or attribute) in a cell in the first column
+        Accessibility API Mappings 1.0 [[html-aam-1.0]] specification.
+      </p>
+      <p>
+        Each language feature (element or attribute) in a cell in the first column
         implies the ARIA semantics (any role, states, and properties) given in
         the cell in the second column of the same row. The third cell in each
         row defines which ARIA role values and aria-* attributes which MAY be
         used. Where a cell in the third column includes the term
         <dfn><strong>Any</strong> `role`</dfn> it indicates that any `role`
         value apart from the <a>implicit ARIA semantics</a> `role` value, MAY
-        be used.
+        be used. If an cell in the third column includes the term <dfn><strong>No `role`</strong></dfn> it indicates that authors SHOULD NOT overwrite the implicit ARIA semantics, or native semantics of the HTML element.
```

Comment:
MUST NOT

Reply:
more simply it means that the role attribute is not allowed on the element

Reply:
MUST it is :)